### ISSUE TITLE
Multiline comment formatting broken by table

### ISSUE THREAD

**Issue by gji** [PeterWone/vsc-print#78]
Multiline comments aren't displayed properly, e.g.:

![multiline](https://user-images.githubusercontent.com/4430689/110014551-ed4a5900-7cf0-11eb-8be7-7a76975b3696.png)

The produced markup looks like:

```
<table class="hljs"><tr><td class="line-number">1</td><td class="line-text"><span class="hljs-string">&#x27;&#x27;&#x27;test</td></tr>
<tr><td class="line-number">2</td><td class="line-text">test</td></tr>
<tr><td class="line-number">3</td><td class="line-text">test&#x27;&#x27;&#x27;</span></td></tr></table>
```

I believe the issue is that highlightjs adds the span with class `hljs-string`, but since the text which is inside the class contains newlines the text naturally gets split across table trs. As a result, the span tag gets split across multiple lines, which leads to invalid markup (tags not nested) and the wrong output. This probably affects multiline strings as well.

I did a quick fix of this issue by using pre and span for whitespace formatting and line numbering instead of tables. The resulting markup is:

```
<pre class="hljs"><span class="lineno">0</span><span class="hljs-string">&#x27;&#x27;&#x27;test
<span class="lineno">1</span>test
<span class="lineno">2</span>test&#x27;&#x27;&#x27;</span></pre>
```

Since lines aren't contained inside their own tags, the resulting markup is valid. This is a pretty big change that might break a bunch of other stuff, but if this seems like a good direction I can submit a PR.

**Comment by PeterWone** [comment]
Too much day job pressure to get this the head-space it deserves but your overall approach does seem like it will bear fruit. If you can get it to fly we can see what if any collateral damage this produces. I have some hope that we can integrate this.

**Comment by gji** [comment]
Sorry, also been a bit busy lately. The fix I implemented above leads to code wrapping not being properly indented and also to manually set the column width for the line numbers. I think the second issue is fixable (possibly by adding a hidden "000" to each line number span) but the first one is a bit of a dealbreaker - I can't think of a fix besides adding JavaScript to find the wrapping locations and manually add in indentation. What may be better is simply finding all instances of span tags that wrap across multiple lines and adding in extra ones, which is what's done e.g. [here](https://github.com/wcoder/highlightjs-line-numbers.js/issues/21) - I can try to do something like that instead.

**Comment by PeterWone** [comment]
I've been pretty busy too. Are you having any success? I've had a new feature by PR and closed a bug myself (poor initial stylesheet) so I'll be doing a release soon. I haven't yet updated the documentation so you do have some time. Do you think you are likely to get this sorted in the next week or so?

**Comment by gji** [comment]
Yeah, I have some free time finally - I'll try to put something together, should be doable in the next few days.

**Comment by gji** [comment]
Should be fixed by #85.
